app/[locale]/student-activities/scroll-aware-section.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import { type ReactNode } from 'react';
```

Comment:
space in between please.

Reply:
can't we just auto format these, things like these are really easy to miss

Reply:
Not sure how that's done. I haven't checked.